ecs/create-resource/main.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_ecs_task_definition" "service" {
+  family                = "${var.family}"
+  container_definitions = "${var.container_definitions}"
+  task_role_arn = "${var.task_role_arn}"
+}
+
+resource "aws_lb_target_group" "target_group" {
+  name     = "${var.service_name}"
+  port     = 80
+  protocol = "HTTP"
+  vpc_id   = "${var.vpc_id}"
+  health_check {    
+    healthy_threshold   = 5    
+    unhealthy_threshold = 2    
+    timeout             = 5    
+    interval            = 30    
+    path                = "${var.health_check_path}"    
+    port                = "traffic-port"
+    matcher             = "200-299"
```

Comment:
Assuming any 2xx response is good. Our health checks currently return 204s when they're happy, though.